src/main.ts

```diff
@@ -0,0 +1,97 @@
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+import { IssuesAddLabelsParams, PullsUpdateParams } from '@octokit/rest';
+
+import {
+  pivotal,
+  getHotfixLabel,
+  addLabels,
+  getPodLabel,
+  filterArray,
+  isBotPr,
+  updatePrDetails,
+  getPivotalId,
+  getPrDescription,
+} from './utils';
+
+async function run() {
+  try {
+    const PIVOTAL_TOKEN: string = core.getInput('pivotal-token', { required: true });
+    const GITHUB_TOKEN: string = core.getInput('github-token', { required: true });
+
+    const {
+      payload: { repository, organization, pull_request },
+    } = github.context;
+
+    let headBranch: string = '';
+    let baseBranch: string = '';
+    if (pull_request && pull_request.base && pull_request.head) {
+      baseBranch = pull_request.base.ref || '';
+      headBranch = pull_request.head.ref || '';
+    }
+    if (!headBranch && !baseBranch) {
+      core.setFailed('Unable to get the head and base branch');
+      process.exit(1);
+    }
+
+    console.log('Base branch -> ', baseBranch);
+    console.log('Head branch -> ', headBranch);
+
+    if (isBotPr(headBranch)) {
+      console.log('This is an automated PR so ignoring rest of the checks.');
```

Comment:
[random]

```suggestion
      console.log('You look like a bot 🤖 so we're letting you off the hook!');
```

Reply:
sorry @rheaditi not committing this. if I commit then I have to pull the changes in local, build the `lib` files and push again. if we had other comments to fix. I would have done this alongside those changes. I'll remember this and push it with the next set of changes.